Dockerfile

```diff
@@ -2,10 +2,15 @@
 FROM golang:latest as builder
 WORKDIR /go/src/github.com/PierreZ/goStatic
 COPY . .
-RUN CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -tags netgo -installsuffix netgo
+RUN mkdir ./bin && \
+    CGO_ENABLED=0 GOARCH=amd64 GOOS=linux go build -tags netgo -installsuffix netgo -o ./bin/goStatic && \
+    mkdir ./bin/etc && \
+    cp ./passwd ./bin/etc && \
```

Comment:
Is it possible to generate those files instead of copying? I think it is better to have another proper stage where we will cleanup users and create a unique one

Reply:
Absolutely. I'll take a look when I get back to my desk.